manifests/charts/gateways/istio-ingress/templates/meshexpansion.yaml

```diff
@@ -37,6 +46,15 @@ spec:
         host: istiod.{{ .Values.global.istioNamespace }}.svc.{{ .Values.global.proxy.clusterDomain }}
         port:
           number: 15012
+  {{- if  .Values.global.centralIstiod }}
+  - match:
+    - port: 443
+    route:
+    - destination:
+        host: istiod.istio-system.svc.cluster.local
```

Comment:
better to not hard code the host name, replace this with:
```
istiod.{{ .Values.global.istioNamespace }}.svc.{{ .Values.global.proxy.clusterDomain }}
```

Reply:
Thanks, updated!